src/main/java/uk/gov/justice/digital/delius/jpa/standard/entity/ManagementTierId.java

```diff
@@ -0,0 +1,30 @@
+package uk.gov.justice.digital.delius.jpa.standard.entity;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.Column;
+import javax.persistence.Embeddable;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import java.io.Serializable;
+import java.time.LocalDateTime;
+
+@Embeddable
+@AllArgsConstructor
+@NoArgsConstructor
+@Builder
+@Data
+public class ManagementTierId implements Serializable {
+    @Column(name = "OFFENDER_ID")
+    private Long offenderId;
+
+    @ManyToOne
+    @JoinColumn(name = "TIER_ID")
+    private StandardReference tier;
+
+    @Column(name = "DATE_CHANGED")
+    private LocalDateTime dateChanged;
```

Comment:
So I guess these are never updated, we just add new ones as they change? And Delius uses the latest?

Reply:
No, there aren't any updates to this table as I understand it. It looks like it provides a history of tier changes. 

I am not clear what Delius does downstream. All the values we're writing are at the instruction of Michael Wetherall. 

This table is pretty odd - no primary key column and half the columns are never used